example_test.go

```diff
@@ -6,10 +6,161 @@ package netaddr_test
 
 import (
 	"fmt"
+	"os"
+	"text/tabwriter"
 
 	"inet.af/netaddr"
 )
 
+func ExampleIP() {
+	ip, err := netaddr.ParseIP("192.0.2.3")
+	if err != nil {
+		panic(err)
```

Comment:
Nit: we have MustParseIP, should we just use that?

Reply:
I purposely chose `ParseIP`, because this is an example of how to use the library in a non-testcase context (e.g. an app). Readers may not reliase that `MustParse` can panic.